src/styles/block.scss/_clock.scss

```diff
@@ -0,0 +1,126 @@
+.clock {
+  position: absolute;
+  height: $size;
+  width: $size;
+  border: $frame-thickness solid $first-color;
+  border-radius: 100%;
+  left: $nule;
+  top: $nule;
+  right: $nule;
+  bottom: $nule;
+  margin: auto;
+
+  &__fon {
+    background-image: url(/styles/block.scss/aaa1.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    opacity: 0.7;
+    height: $background-size;
+    width: $background-size;
+    border-radius: 100%;
+    position: absolute;
+    left: $nule;
+    top: $nule;
+    right: $nule;
+    bottom: $nule;
+    margin: auto;
```

Comment:
There is no need to create a variable for each property)
Create them only for most important\frequent values (colors, main sizes, durations etc)
![image](https://user-images.githubusercontent.com/96409563/224283178-bf112dbc-5b5f-4039-b604-f4a453f04985.png)